.circleci/config.yml

```
@@ -355,6 +366,9 @@ workflows:
           filters:
             branches:
               ignore: main
+      - pylint:
+          requires:
+            - pip-install
```

```suggestion
            - install
```
The current workflow is not run because of a configuration error:

> Job 'pylint' requires 'pip-install', which is the name of 0 other jobs in workflow 'develop'

– https://app.circleci.com/pipelines/github/digitalfabrik/lunes-cms?branch=feature%2Fadd_pylint